modules/simple-payments/simple-payments.php

```diff
@@ -62,9 +62,9 @@ public function init_hook_action() {
 
 	function register_gutenberg_block() {
 		if ( $this->is_enabled_jetpack_simple_payments() ) {
-			jetpack_register_block( 'simple-payments' );
+			register_block_type( 'jetpack/simple-payments' );
```

Comment:
Same as above, I think we should check for the `register_block_type` before to use it. Another alternative would be to wrap all those calls in `if ( Jetpack_Gutenberg::is_gutenberg_available() )` checks, like we do for the Contact Form.

Reply:
Added the check per 3f93ab58b